src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -52,18 +70,24 @@ export class ChoiceGroup extends React.Component<IChoiceGroupProps, IChoiceGroup
     return (
       // Need to assign role application on containing div because JAWS doesnt call OnKeyDown without this role
       <div role='application' className={ className }>
-        <div className='ms-ChoiceFieldGroup' role='radiogroup'  aria-labelledby={ this.props.label ? this._id + '-label' : '' }>
+        <div
+          className='ms-ChoiceFieldGroup'
+          role='radiogroup'
```

Comment:
radiogroup implies only 1 item is checked at a time. In addition, the children should be role="radio". I think what you want is role="combobox" here, and children of role="option"